src/client/features/admin/index.js

```diff
@@ -7,7 +7,7 @@ const classNames = require('classnames');
 
 const Icon = require('../../common/components').Icon;
 const PathwayCommonsService = require('../../services').PathwayCommonsService;
-const CDC = require('../../services/index.js').CDC;
+const CDC = require('../../services/index').CDC;
```

Comment:
@jonahdlin  @gaelder @hmtinc if you are requiring directories and there is an index.js file in that directory, it requires that automatically.

e.g
```js
const CDC = require('../../services/index').CDC;
// is equivalent to
const CDC = require('../../services/').CDC;
```